content/assets/css/application.scss

```diff
@@ -83,11 +84,6 @@
   padding: 0px;
 }
 
-.govuk-grid-column-one-half {
```

Comment:
We shouldn't making all half-columns flex. I created `dfe-card-container` class to do this job.